tests/unit/vertexai/test_any_serializer.py

```diff
@@ -47,6 +47,8 @@
 
 try:
     import bigframes as bf
+    if bf.__version__ >= "1.0.0":
+        raise ImportError("BigFrames version is greater than or equal to 1.0.0. Please install a version less than 1.0.0.")
 except ImportError:
```

Comment:
the error on line 51 will be caught by 52 without logging

Reply:
if bf is set to None it will throw an error here: https://github.com/googleapis/python-aiplatform/blob/67a10e98bd2a55a871bcdfbbc608b96a89d546aa/tests/unit/vertexai/test_any_serializer.py#L313

Reply:
added a print statement so its verbose.